testdata/go/memcached-operator/api/v1alpha1/memcached_webhook.go

```diff
@@ -17,7 +17,6 @@ limitations under the License.
 package v1alpha1
 
 import ("errors"
-	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
```

Comment:
This change in the scaffold is because of the bugfix `create webhook` subcommand for the defaulting webhooks by removing import that is not used. More info: https://github.com/kubernetes-sigs/kubebuilder/pull/1718